packages/core/package.json

```
@@ -6,7 +6,7 @@
   "private": true,
   "scripts": {
     "start": "npm run test && npm run serve",
-    "serve": "echo \"Serving...\" && sudo ./node_modules/.bin/webpack-dev-server --config \"./build/webpack.config.js\"",
+    "serve": "f() { echo \"Serving...\" && sudo ./node_modules/.bin/webpack-dev-server --config \"./build/webpack.config.js\" --env.package $1;}; f",
```

Following the previous comment:
```suggestion
    "serve": "f() { echo \"Serving...\" && sudo ./node_modules/.bin/webpack-dev-server --config \"./build/webpack.config.js\" --env.open $1;}; f",
```